content/en/blog/2025/stability-proposal-announcement.md

```diff
@@ -0,0 +1,181 @@
+stable. Stability requirements will also be streamlined and clarified for
+contributors, maintainers, and end-users. We plan to expand these stability
+requirements to include documentation, examples, implementation scenarios,
+performance benchmarks, and other useful information.
+
+In addition, we are proposing a change to the defaults of the SDK and
+instrumentation agents. By default, stable components should not automatically
+import, include, or enable unstable components. For example, if you install a
+stable distribution of OpenTelemetry Java Instrumentation Agent, the default
+receivers, processors, samplers, exporters, and libraries should not include
+alpha or experimental instrumentations.
+
+Similarly, the OpenTelemetry JavaScript Node Automatic Instrumentation
+Metapackage would only include stable instrumentations (e.g., those which are at
+v1.0 or better). For consistency, we propose a single project-wide way to enable
+unstable behavior for end-users. The goal of this proposal is to reduce the
+likelihood of end-user adoption of unstable packages, components, config, or API
+surface area without an explicit ‘opt-in’ to this behavior.
+
+We do not make this change lightly, and it wasn’t feasible to make it sooner –
+end-user feedback has been a crucial aspect of our development process, and
+early adoption of unstable instrumentation and features has led to better
+outcomes overall. However, given the scope of OpenTelemetry it can be difficult
+for users to evaluate individual component stability when installing
+dependencies. By moving to a ‘default stable’ approach, we can ensure that users
+will not be surprised by unexpected changes.
+
+## Adjusting stability requirements for components
+
+Second, we are adjusting our overall posture when it comes to instrumentation
+and instrumentation stability. Our goal, as a project, has always been to see
+other libraries, frameworks, runtimes, and tools
+[natively adopt OpenTelemetry](https://www.youtube.com/watch?v=l8xiNOCIdLY).
+This is still the goal – we want other developers to embed our APIs into their
+software, rather than requiring you to download and install libraries that
+inject instrumentation code into third party dependencies. However, many users
+have come to see our ‘contrib’ repositories and distributions as generic
+replacements for system telemetry agents.
+
+We’re thrilled that we’ve had so many users adopt the Collector as a
+vendor-agnostic solution for collecting telemetry data, and we believe that it’s
+a strong testament to the long-term viability of the project. What has become
+challenging is that our contrib repositories – especially in the Collector –
+represent a huge amount of work for maintainers and a constant drag on project
+velocity. Even in a world where we had twice as many contributors and
+maintainers, it would still be a challenge to accept all desired receivers or
+instrumentation libraries and ensure they all met a sufficiently high quality
```

Comment:
I'm not sure if instrumentation libraries should be related to the collector work. I might be wrong about this, but it doesn't seem like the collector expertise transfers to the instrumentation libraries, ie, a person able to make a mysql receiver is not actually the best at building a python instrumentation library, even on mysql.